scripts/start-finalExec

```diff
@@ -221,7 +221,11 @@ function copyFilesForCurseForge() {
   cp -f /data/eula.txt "${FTB_DIR}/"
 }
 
-if versionLessThan 1.7; then
+if versionLessThan 'b1.8'; then
```

Comment:
Re-reading the protocol docs again, I think the existing condition for < 1.7 is still correct given https://wiki.vg/Server_List_Ping#1.6, so please put that back as an `elif`

...and before that is where you could add a new condition for < 1.6 which will disable the health check...for now.

Also with re-reading the docs, I realized one could also improve the implementation in https://github.com/itzg/mc-monitor/blob/master/slp/slp.go to handle the 1.4-1.5 and the <=1.3 variations of the protocol messages.

Reply:
The < 1.7 is still in there in line 228.

The check for 1.6 still works with older versions from b1.8 - 1.5 as the sent data is only additive.
I just tested it again with 1.4.7 and the existing healthcheck works without any Problem.
You can just use the 1.6 healthcheck for that.

Reply:
🤦‍♂️sorry, I missed the `elif` on line 228.